test/runtime/samples/inline-style-directive-multiple/_config.js

```diff
@@ -0,0 +1,16 @@
+export default {
+	html: `
+	<p style="color: red; width: 65px; font-weight: 700;"></p>
+	`,
+
+	test({ assert, component, target, window }) {
+		const p = target.querySelector('p');
+
+		const styles = window.getComputedStyle(p);
+		assert.equal(styles.color, 'red');
+	},
+
+	skip_if_ssr: true
+	// SSR renders "null" string for null values:
+	// <p style="color: red; width: 65px; position: null; font-weight: 700;"></p>
```

Comment:
i guess you need to fix the case for SSR when the value is null, do not add the css property to the style?

Reply:
So, I was going to do this, until I realized that the current behavior will render `property: null` styles when doing SSR, e.g. with:

```svelte
<script>
	let color = null;
</script>
<h1 style="color: {color}">Hello!</h1>
```

Svelte generates:
```js
let color = null;
const App = create_ssr_component(($$result, $$props, $$bindings, slots) => {
	return `<h1 style="${"color: " + escape(color)}">Hello!</h1>`;
});
```  

Which means the output is:
```html
<h1 style="color: null">Hello!</h1>
```

Which is maybe not great? Or maybe fine?

The thing is, to check null values, I think we'd have to do something that ends up looking like this:

```js
const App = create_ssr_component(($$result, $$props, $$bindings, slots) => {
	return `<h1 style="${(isRed ? "red" : null) ? "color: " + escape((isRed ? "red" : null)) : ""}">Hello!</h1>`;
});
```

Reply:
i've updated the code that handles the `null` case